app/containers/SearchPage/SearchView.tsx

```diff
@@ -495,7 +465,7 @@ class SearchView extends React.PureComponent<SearchViewProps> {
           removeSearchTerm,
           this.props.onUpdateParams,
         )}
-        page={this.props.params.page}
+        page={Math.min(this.props.params.page, pagesTotal)}
```

Comment:
Make it so you can't go past the pageTotal (which happened sometimes when you switched searches)